README.md

```diff
@@ -148,11 +148,46 @@ You might want to add that command to your `npm test` script in your `package.js
 By default the Fastboot app will be created with the same Ember.js version you have specified in your addon's `bower.json`. But you can override this with an additional option:
 
     ember fastboot:test --ember-version <version>
-    
+
 You can use any version you would also specify in your `bower.json`, e.g. `release` or `2.4.0`. *Note that Fastboot itself requires at least Ember.js 2.4!*
-    
+
 See `ember help fastboot:test` for additional options.
 
+### Working with `ember-try`
+
+By default, addons use [`ember-try`][ember-try] to run tests against multiple versions of dependencies simultaneously on Travis CI.  This system can also be leveraged to run your Fastboot tests in parallel with your browser tests.  I just takes two small changes:
```

Comment:
"I just" should probably be "It just"!

Apart from that thanks a lot for putting this together! 👍